doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -164,20 +163,20 @@ typedef returntype (*functiontype)(argumenttype, ...)
 ```
 
 The macro [`@cfunction`](@ref) generates the C-compatible function pointer for a call to a
-Julia function. Arguments to [`@cfunction`](@ref) are as follows:
+Julia function. The arguments to [`@cfunction`](@ref) are:
 
 1. A Julia Function
-2. Return type
+2. The return type
```

Comment:
That depends on how you interpret it.  If you think about the return type as belonging to the function, then I think it should be `the function's return type` (which I shortened to `the return type` here), since the function was introduced above.  If it's an arbitrary return type, then `a return type` would be better.

I'll change it to `the function's return type`, since that's clearer.

Reply:
OK, I guess I was reading it as "A call's return type"